.github/workflows/ci-cd.yml

```diff
@@ -144,6 +144,8 @@ jobs:
             python-version: "3.9"
           - image-builder-version: "2024.10"
             python-version: "3.13"
+          - image-builder-version: "PREVIEW"
+            python-version: "3.13"
```

Comment:
Should probably also test these on 3.9 (could be possible that we pin a new version that doesn't work with Python 3.9 (requires some additional transitive dependency, or drops support altogether).